docs/GroupDefs.hpp

```diff
@@ -48,6 +48,49 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between frames, such as
+ * the fixed computational frame (grid frame) and the inertial frame. The
+ * time-dependent parameters of the mapping are adjusted by a feedback control
+ * system in order to follow the dynamical evolution of the black holes. In BBH
```

Comment:
evolution of objects such as horizons of black holes, surfaces of neutron stars, ....  For example, in binary black hole